client/my-sites/posts/posts-navigation.jsx

```diff
@@ -134,6 +134,7 @@ export default React.createClass( {
 				}
 				<Search
 					pinned={ true }
+					siteID = { this.props.siteID }
```

Comment:
Ditto as above: maybe remove the spaces around the `=` sign to match the other props?